src/display/text_layer.js

```diff
@@ -46,23 +47,43 @@ const DEFAULT_FONT_SIZE = 30;
 const DEFAULT_FONT_ASCENT = 0.8;
 const ascentCache = new Map();
 
-function getAscent(fontFamily, ctx) {
+function getCtx(size, isOffscreenCanvasSupported, alpha = true) {
+  let ctx;
+  if (isOffscreenCanvasSupported && FeatureTest.isOffscreenCanvasSupported) {
+    ctx = new OffscreenCanvas(size, size).getContext("2d", { alpha });
```

Comment:
Do we "need" to use `isOffscreenCanvasSupported` here, i.e. are the any clear benefits?
If not, maybe do that in a separate commit to reduce the size of these changes such that reviewing becomes slightly easier?

Reply:
Please move your question here: https://github.com/mozilla/pdf.js/pull/15722 and I'll answer.